cfme/tests/intelligence/reports/test_crud.py

```diff
@@ -264,3 +266,21 @@ def test_reports_delete_saved_report(appliance, request):
     view.configuration.item_select(
         item='Delete selected Saved Reports', handle_alert=True)
     assert not report.exists
+
+
+def test_reports_create_schedule_for_custom_report(
+    appliance, request, custom_report_values, schedule_data
+):
+    custom_report = appliance.collections.reports.create(**custom_report_values)
+    schedule_data["filter"] = (
+        "My Company (All Groups)",
+        "Custom",
+        custom_report.menu_name,
+    )
+    try:
+        custom_report_schedule = appliance.collections.schedules.create(**schedule_data)
+    except (MoveTargetOutOfBoundsException, NoSuchElementException):
+        custom_report_schedule = appliance.collections.schedules.create(**schedule_data)
```

Comment:
Question : 
Your try and except both are doing the same thing??
custom_report_schedule = appliance.collections.schedules.create(**schedule_data)

Reply:
Yes. The command fails the first time. Locally it gives me a `NoSuchElementException` and on PRT it gives `MoveTargetOutOfBoundsException`. I don't understand why, but the command works fine when I run it again.

Reply:
Okay please write a comment about it then .
It's only through code that this command fails for the first time right ?
If in UI also this happens then it might be a BZ , something to investigate .